scripts/configurator/parser/cmake_row.py

```diff
@@ -9,20 +23,22 @@ class cmake_row:
    either file_lines or an array of cmake_row. Among these 2 it make more sense
    to just keep file_lines.
    """
-   EMPTY: str = "EMPTY"
-   SLASH_COMMENT :str = "SLASH_COMMENT"
-   HASHTAG_COMMENT: str = "HASHTAG_COMMENT"
-   VARIABLE: str = "VARIABLE"
-   NOT_PARSABLE: str = "NOT_PARSABLE"
    regex_expression: str =  r"^([A-Za-z_0-9]+?):([A-Za-z]+?)=(.*)$"
 
-   BOOL_MAPPINGS = {
-      "ON" : "ON",
-      "OFF": "OFF",
-      "0" : "OFF",
-      "1" : "ON",
-      "TRUE": "ON",
-      "FALSE": "OFF",
+   NORMALIZED_BOOLS = {
+      "ON" : True,
+      "OFF": False,
+      "1" : True,
+      "0" : False,
+      "TRUE": True,
+      "FALSE": False,
+   }
+
+   STR_TO_ENUM  = {
```

Comment:
Here this shouldn't be STR_TO_ENUM. This should be something like CMakeTypeToConfiguratorTypeMap.

Reply:
Changed.